src/modules/renderGraphiQL.ts

```diff
@@ -95,7 +95,7 @@ export function renderGraphiQL(data: GraphiQLData): string {
       }
     }
     // We don't use safe-serialize for location, becuase it's not client input.
-    var fetchURL = locationQuery(otherParams, ${endpointURL});
+    var fetchURL = locationQuery(otherParams, '${endpointURL}');
```

Comment:
Yeah, this one seems like a no-brainer. I'm pretty sure that wouldn't have worked for me. I think it worked when I called the safe-serialize, but then I removed it and probably didn't test again.